msg/Gpzda.msg

```diff
@@ -0,0 +1,16 @@
+# Message from GPRMC NMEA String
+Header header
+
+string message_id
+
+float64 utc_seconds
```

Comment:
Is this ever floating point? Both of the references seem to indicate that it's integer, and the meaning of the message (time of the pulse-per-second) also seems to suggest that the resolution is full-seconds.

Reply:
This is likely to be an int32. I used float64 so it is consistent with utc_seconds in other messages. It this is not necessary, I will update it.